packages/graphback-schema/src/plugin.ts

```diff
@@ -1,15 +1,13 @@
-import { inputTypeVisitor } from 'graphback'
+import { GraphbackContextCreator, InputContext } from 'graphback'
 import { DocumentNode, GraphQLSchema, parse, printSchema, visit } from 'graphql';
 import { generateSchema } from './schemaTemplate';
-import { buildTargetContext, InputContext, TargetContext } from './targetType';
+import { buildTargetContext, TargetContext } from './targetType';
 
 export const plugin = (schema: GraphQLSchema) => {
-  const printedSchema: string = printSchema(schema)
-  const astNode: DocumentNode = parse(printedSchema)
-  
-  const inputTypes = visit(astNode, { leave: inputTypeVisitor })
 
-  const context: TargetContext = buildTargetContext(inputTypes.definitions.reverse())
+  const inputContext = new GraphbackContextCreator(printSchema(schema)).createInputContext()
```

Comment:
We are initializing the Input Context here. Current way using visitor pattern doesn't require using codegen plugin as we can pass this Input Context to different use cases. 

Codegen further needs to accept schema instance everytime which is really not required.

@wtrocki  WDYT?